.github/workflows/build.yaml

```diff
@@ -143,3 +146,47 @@ jobs:
           APICULTURIST_GITHUB: ${{ secrets.APICULTURIST_GITHUB }}
           APICULTURIST_TOKEN: ${{ secrets.APICULTURIST_TOKEN }}
           monorepo: ${{ inputs.monorepo }}
+
+  dev_build:
+    runs-on: ubuntu-latest
+    needs: read_package
+    name: ${{ matrix.repos }}
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ${{fromJSON(needs.read_package.outputs.repoNames)}}
+    steps:
+      - name: Setup workspace
+        uses: qlik-oss/sn-gh-workflows/actions/setup@v3.0.1
+        with:
+          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN || secrets.NPM_CI_TOKEN }}
+          NPMRC_ENCODED: ${{ secrets.QLIK_NPM_DEV_ENCODED }}
+      - name: Dev build
+        run: pnpm -r --filter ${{ matrix.repos }} build:dev
+      - name: Upload dev build
+        uses: qlik-trial/qmfe-workflows/actions/qmfe-to-aws@v1
+        with:
+          aws-access-key-id: ${{ secrets.QMFE_DEV_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.QMFE_DEV_ACCESS_KEY }}
+          qmfe-id: ${{ matrix.repos }}
+          source: ${{fromJSON(needs.read_package.outputs.repoFolders)[strategy.job-index]}}/dist
+          version: ${{ github.event.number }}
+          allow-overwrite: true
+          include-sourcemaps: true
+          aws-bucket-name: "hub-devbuilds"
+          s3-key: "nebula"
```

Comment:
Not sure if this needs to be nebula or if qmfe is also okay

Reply:
How can answer that? Anders

Reply:
This seems to work. I think it only manages the folder. I checked the bucket itself which needs to be hub-devbuilds at least.